### ISSUE TITLE
Pretrain [WARNING:root:NaN or Inf found in input tensor.]

### ISSUE THREAD

**Issue by ghost** [MhLiao/MaskTextSpotter#8]
@MhLiao Thank you for your hard work
My config file is attached: [pretrain.yaml.zip](https://github.com/MhLiao/MaskTextSpotter/files/3749111/pretrain.yaml.zip)

When trying to train ICDAR2015 using Pretrain, I keep getting `NaN` errors.

My training command:
```
python3 -m torch.distributed.launch --nproc_per_node=1 tools/train_net.py --config-file configs/pretrain.yaml
```
INFO:maskrcnn_benchmark.trainer:
```
loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_mask: nan (nan)  loss_char_mask: 0.0000 (0.0000)  loss_seq: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: nan (nan)
```

The errors that I keep getting:
```
WARNING:root:NaN or Inf found in input tensor.
```

**Comment by MhLiao** [comment]
@deepseek Can you train the model with ICDAR2013? I have no idea about this error currently.

**Comment by mrocr** [comment]
hmmmm...
 thats weird, for icdar 2013 pretrain is working fine.
but for icdar 2015 it keeps showing the NaN error.

**Comment by ghost** [comment]
hmmmm...
thats weird, for icdar 2013 pretrain is working fine.
but for icdar 2015 it keeps showing the NaN error.

**Comment by MhLiao** [comment]
@deepseek Did you try to decrease the base learning rate for the icdar2015 dataset. It may be related to the difficulty of the datasets.

**Comment by ghost** [comment]
I changed learning rate to `BASE_LR: 0.001` and it was solved
you are smart, thanks man.
